README.md

```diff
@@ -1 +1,64 @@
-# Infolink
\ No newline at end of file
+| **Package** |**Version** |**Downloads** |
+| --- | --- | --- |
+| `SimplyWorks.Infolink` | [![NuGet](https://img.shields.io/nuget/v/SimplyWorks.Infolink.Sdk)](https://www.nuget.org/packages/SimplyWorks.Infolink.Sdk/) | [![Nuget](https://img.shields.io/nuget/dt/SimplyWorks.Infolink.Sdk)](https://nuget.org/packages/SimplyWorks.Infolink) |
+
+
+[![Build Status](https://dev.azure.com/simplify9/Github%20Pipelines/_apis/build/status/simplify9.Infolink?branchName=master)](https://dev.azure.com/simplify9/Github%20Pipelines/_build/latest?definitionId=168&branchName=master) ![Azure DevOps tests](https://img.shields.io/azure-devops/tests/Simplify9/Github%20Pipelines/168?) [![Generic badge](https://img.shields.io/badge/contributions-WELCOME-<green>.svg)](https://shields.io/)
+
+## Infolink 
+
+Infolink is an abstracted message handling framework, that makes it extremely convenient to
+add message handling into an existing project. Infolink sits on top of a queue and filters it into
+different **Documents**, passing them into different **Subscribers** where each **Subscriber** is
+directly related to a **Partner**.
+
+### Documents
+
+A **Document** is essentially a message that can be correlated with a *class* name. Whenever it
+is published using the `Publisher` from [SW.Bus](https://github.com/simplify9/Bus/) that can be
```

Comment:
IPublish is for the bus msg based documents only, there are 4 different documents types